dev/NavigationView/NavigationView.cpp

```diff
@@ -1600,7 +1600,7 @@ void NavigationView::OnKeyDown(winrt::KeyRoutedEventArgs const& e)
     switch (key)
     {
     case winrt::VirtualKey::GamepadView:
-        if (!IsPaneOpen())
+        if (!IsPaneOpen() && !IsTopNavigationView())
```

Comment:
Is it possible to add test case? There are a some existing test cases which is using gamepad in ScrollViewerTestsWithInputHelper.cs.

Reply:
I have added the appropriate test case.